core/src/main/scala/kanaloa/reactive/dispatcher/Dispatcher.scala

```diff
@@ -144,7 +146,8 @@ object PushingDispatcher {
     backend:    T,
     rootConfig: Config = ConfigFactory.load()
   )(resultChecker: ResultChecker)(implicit system: ActorSystem) = {
-    val (settings, metricsCollector) = Dispatcher.readConfig(name, rootConfig)
+    val (settings, reporter) = Dispatcher.readConfig(name, rootConfig)
```

Comment:
Yep, file this comment as when I was still confused bout the difference between internal/external metrics